apps.md

```diff
@@ -40,6 +40,12 @@ mentioned. Please note that other apps can be forced to use it by following
 | SSH | [OpenSSH-281.40.2](https://github.com/apple-oss-distributions/OpenSSH) | `applemultipath` option |
 | [Shellfish](https://secureshellfish.app) | ? | Settings / Advanced / 🌍 Multipath TCP |
 
+## iOS apps
+
+| Name | Version |
+| --- | --- |
```

Comment:
(do you know if there are other apps we can add here? Might be good to add Siri, Apple Music, and Apple Maps in fact)

Reply:
Other than the apps published by Apple, I don't know any sadly